lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -89,10 +89,43 @@ def custom_build_settings
             settings
           end
 
+          # @param [Generator] generator
+          #        the generator to use for generating the content.
+          # @param [Pathname] path
+          #        the pathname to save the content into.
+          #
+          # Saves the content the provided path unless the path exists and the contents are exactly the same.
+          #
+          # @return [Void]
+          #
+          def update_changed_file(generator, path)
+            if File.exist?(path)
```

Comment:
@segiddins sorry for the `support_files_temp_dir.dir?` which line method do you say it should be replaced?